clients/src/main/resources/common/message/DescribeQuorumResponse.json

```diff
@@ -26,50 +26,57 @@
       "about": "The top level error code."},
     { "name": "ErrorMessage", "type": "string", "versions": "2+", "nullableVersions": "2+", "ignorable": true,
       "about": "The error message, or null if there was no error." },
-    { "name": "Topics", "type": "[]TopicData",
-      "versions": "0+", "fields": [
+    { "name": "Topics", "type": "[]TopicData", "versions": "0+",
+      "about": "The response from the describe quorum API.", "fields": [
       { "name": "TopicName", "type": "string", "versions": "0+", "entityType": "topicName",
         "about": "The topic name." },
-      { "name": "Partitions", "type": "[]PartitionData",
-        "versions": "0+", "fields": [
+      { "name": "Partitions", "type": "[]PartitionData", "versions": "0+",
+        "about": "The partition data.", "fields": [
         { "name": "PartitionIndex", "type": "int32", "versions": "0+",
           "about": "The partition index." },
-        { "name": "ErrorCode", "type": "int16", "versions": "0+"},
+        { "name": "ErrorCode", "type": "int16", "versions": "0+",
+          "about": "The partition error code."},
         { "name": "ErrorMessage", "type": "string", "versions": "2+", "nullableVersions": "2+", "ignorable": true,
           "about": "The error message, or null if there was no error." },
         { "name": "LeaderId", "type": "int32", "versions": "0+", "entityType": "brokerId",
           "about": "The ID of the current leader or -1 if the leader is unknown."},
         { "name": "LeaderEpoch", "type": "int32", "versions": "0+",
-          "about": "The latest known leader epoch"},
-        { "name": "HighWatermark", "type": "int64", "versions": "0+"},
-        { "name": "CurrentVoters", "type": "[]ReplicaState", "versions": "0+" },
-        { "name": "Observers", "type": "[]ReplicaState", "versions": "0+" }
+          "about": "The latest known leader epoch."},
+        { "name": "HighWatermark", "type": "int64", "versions": "0+",
+          "about": "The high watermark of the partition."},
```

Comment:
"The high water mark" would be better. Unfortunately, it should be "HighWaterMark" too, but that mistake is too late to fix.